articles/app-service-mobile/app-service-mobile-xamarin-forms-get-started-users.md

```diff
@@ -219,14 +246,25 @@ This section shows how to implement the **IAuthenticate** interface in the iOS a
         }
 
     If you are using an identity provider other than Facebook, choose a different value for [MobileServiceAuthenticationProvider].
+
+6. Add OpenUrl(UIApplication app, NSUrl url, NSDictionary options) method overload
+
+        public override bool OpenUrl(UIApplication app, NSUrl url, NSDictionary options)
+        {
+            return TodoItemManager.DefaultManager.CurrentClient.ResumeWithURL(url);
+        }
+
 6. Add the following line of code to the **FinishedLaunching** method before the call to `LoadApplication()`:
 
         App.Init(this);
 
     This code ensures the authenticator is initialized before the app is loaded.
+
+6. Add {url_scheme_of_your_app} to URL Schemes in Info.plist.
+
 7. Rebuild the app, run it, then sign in with the authentication provider you chose and verify you are able to access data as an authenticated user.
 
-## Add authentication to Windows 8.1 (including Phone) app projects
+## Add authentication to Windows 10 (including Phone) app projects
 This section shows how to implement the **IAuthenticate** interface in the Windows 8.1 and Windows Phone 8.1 app projects. The same steps apply for
 Universal Windows Platform (UWP) projects, but using the **UWP** project (with noted changes). Skip this section if you are not supporting Windows devices.
 
```

Comment:
Project name "WinApp" is renamed to "UWP" already. Change the wording to "In Visual Studio, right-click either the **UWP** project, then **Set as StartUp Project**."